### ISSUE TITLE
"Value definition is empty" Error caused by Class Constant Arrays

### ISSUE THREAD

**Issue by lchambers** [ApiGen/ApiGen#651]
When running the generate command I get a "Value definition is empty" error.

I've traced this back to a single class where I have several class constants that are arrays, eg:

```
const CLASS_CONST = array(
    self::CLASS_CONST_2,
    self::CLASS_CONST_3,
    self::CLASS_CONST_4,
);
```
If I remove all of these then generate runs correctly.

Any ideas as to how to resolve?

For reference I'm running version 4.1.0

**Comment by alexmace** [comment]
We are also getting this error on PHP 5.6 code using class constant arrays.

**Comment by jadb** [comment]
@lchambers have you tried the latest [4.1 branch](https://github.com/ApiGen/ApiGen/tree/4.1)?

**Comment by timcurzon** [comment]
Would it be possible to add a v4.1.x tag to reflect the more recent stable changes on the 4.1 branch? The 4.1.1 tag is fairly out of date.

**Comment by jadb** [comment]
@timcurzon I was supposed to do that last weekend but ran out of time - hopefully getting to that this weekend and tagging 4.2 to start work on 5.0.

**Comment by alexmace** [comment]
Any news on this? We've disabled ApiGen and will probably switch away if this isn't fixed soon...

**Comment by co8bit** [comment]
me too, Waht any news?

**Comment by alexmace** [comment]
Pretty sure this project is dead, migrate away.

Cheers,

Alex

**Comment by kenjis** [comment]
@jadb 
I've got the same error!
I use v4.1.2.

I hope this bug will be fixed soon.

**Comment by POPSuL** [comment]
This problem was fixed by #726
